apps/readest-app/src/components/BookDetailModal.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Book } from '@/types/book';
+
+const BookDetailModal = ({ isOpen, onClose, book }: { isOpen: boolean; onClose: () => void; book: Book }) => {
+  if (!isOpen) return null;
+
+  return (
+    <div className="fixed inset-0 z-50 flex items-center justify-center">
+      {/* Transparent gray overlay */}
+      <div 
+        className="fixed inset-0 bg-gray-800 bg-opacity-70" 
+        onClick={onClose}
+      />
+      
+      <div className="relative z-50 w-96 bg-white rounded-lg shadow-xl p-6">
+        {/* Close button */}
```

Comment:
Better to reuse the `WindowButtons` as this:
https://github.com/chrox/readest/blob/main/apps/readest-app/src/app/reader/components/settings/SettingsDialog.tsx#L78-L83